packages/dd-trace/src/profiling/profilers/space.js

```diff
@@ -1,17 +1,47 @@
 'use strict'
 
+const path = require('node:path')
+const { SnapshotKinds, OOMExportStrategies } = require('../constants')
+
+function strategiesToCallbackMode (strategies, CallbackMode) {
+  return (strategies.includes(OOMExportStrategies.InteruptCallback) ? CallbackMode.Interrupt : 0) |
+    (strategies.includes(OOMExportStrategies.AsyncCallback) ? CallbackMode.Async : 0)
```

Comment:
I would break out the branches into separate variables for more clarity on what is actually being done here. Something like:

```suggestion
  const hasInterrupt = strategies.includes(OOMExportStrategies.InteruptCallback) ? CallbackMode.Interrupt : 0
  const hasCallback = strategies.includes(OOMExportStrategies.AsyncCallback) ? CallbackMode.Async : 0
  return hasInterrupt | hasCallback
```